TELCODOCS-1840 siteConfigError created to surface more info on error … (#88028)
openshift/openshift-docs: modules/ztp-troubleshooting-ztp-gitops-installation-crs.adoc

+
[source,terminal]
----
$ oc get applications.argoproj.io -n openshift-gitops clusters -oyaml
----

> .. Check for the `status.operationState.syncResult.resources` field to view the error logs for the managed cluster. For example, setting an invalid value for `extraManifestPath` in the `SiteConfig` CR raises the following error::

Reviewer: extra `:` at the end.

+
[source,text]
----
syncResult:
  resources:
  - group: ran.openshift.io
    kind: SiteConfig
    message: The Kubernetes API could not find ran.openshift.io/SiteConfig for
      requested resource spoke-sno/spoke-sno. Make sure the "SiteConfig" CRD is
      installed on the destination cluster
----

> .. To look for a more detailed `SiteConfig` error, use the ArgoCD Dashboard. Click the `siteconfig` resource that ArgoCD is trying to sync and then check the `DESIRED MANIFEST` tab for the `siteConfigError` field.

Reviewer 2: ```suggestion
.. To see a more detailed `SiteConfig` error, use the Argo CD dashboard. Click the `SiteConfig` resource that Argo CD is trying to sync and then check the `DESIRED MANIFEST` tab to find the `siteConfigError` field.
```

- Per my understanding `ArgoCD` is a resource, hence to use a plain name I suggest using "Argo CD".
- Since these are the steps on the dashboard/console, should these names in the back ticks be in bold?  -- "**DESIRED MANIFEST**" and "Click the **SiteConfig** resource"

This is optional but you might want to consider breaking steps as follows:
(up to you)

```suggestion
.. To see a more detailed `SiteConfig` error, use the Argo CD dashboard:

  ... Click the *SiteConfig* resource that Argo CD is trying to sync. 
  ... Check the *DESIRED MANIFEST* tab to find the *siteConfigError* field.
```

+
[source,terminal]
----

> $ oc get applications.argoproj.io -n openshift-gitops clusters -oyaml

Reviewer 3: ```suggestion
$ oc get applications.argoproj.io -n openshift-gitops clusters -o yaml
```

----

> .. Check for the `status.operationState.syncResult.resources` field to view the error logs for the managed cluster. For example, setting an invalid value for `extraManifestPath` in the `SiteConfig` CR raises the following error:

Reviewer 3: Explain the purpose of the action before describing the action.
https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#organizing-the-steps-in-a-procedure:~:text=If%20you%20orient%20the%20user%20to%20the%20location%20where%20the%20action%20is%20performed%20or%20explain%20the%20purpose%20of%20the%20action%2C%20do%20so%20before%20describing%20the%20action.

> .. Check for the `status.operationState.syncResult.resources` field to view the error logs for the managed cluster. For example, setting an invalid value for `extraManifestPath` in the `SiteConfig` CR raises the following error:

Reviewer 3: ```suggestion
.. To view the error logs for the managed cluster, check for the `status.operationState.syncResult.resources` field. For example, setting an invalid value for `extraManifestPath` in the `SiteConfig` CR raises the following error:
```

+
[source,text]
----
syncResult:
  resources:
  - group: ran.openshift.io
    kind: SiteConfig
    message: The Kubernetes API could not find ran.openshift.io/SiteConfig for
      requested resource spoke-sno/spoke-sno. Make sure the "SiteConfig" CRD is
      installed on the destination cluster
----

> .. To see a more detailed `SiteConfig` error, use the Argo CD dashboard:

Reviewer 3: ```suggestion
.. To see a more detailed `SiteConfig` error, complete the following steps:
```

Reviewer 3: https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#organizing-the-steps-in-a-procedure:~:text=To%20install%20XYZ%20Manager%2C%20complete%20the%20following%20steps%3A

> ... Click the *SiteConfig* resource that Argo CD is trying to sync. 

Reviewer 3: ```suggestion
... In the Argo CD dashboard, click the *SiteConfig* resource that Argo CD is trying to sync. 
```

Reviewer 3: https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#organizing-the-steps-in-a-procedure:~:text=If%20you%20orient%20the%20user%20to%20the%20location%20where%20the%20action%20is%20performed%20or%20explain%20the%20purpose%20of%20the%20action%2C%20do%20so%20before%20describing%20the%20action.

> ... Check the *DESIRED MANIFEST* tab to find the `siteConfigError` field.

Reviewer 3: ```suggestion
... To find the `siteConfigError` field, check the *DESIRED MANIFEST* tab, for example:
```

Reviewer 3: https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#:~:text=If%20you%20orient%20the%20user%20to%20the%20location%20where%20the%20action%20is%20performed%20or%20explain%20the%20purpose%20of%20the%20action%2C%20do%20so%20before%20describing%20the%20action.
